Lombiq.Tests.UI/SecurityScanning/SecurityScanConfiguration.cs

```diff
@@ -42,7 +42,7 @@ public class SecurityScanConfiguration
     /// for correct error handling. This is achieved by adding the error page URL to the configuration with <see
     /// cref="YamlDocumentExtensions.AddRequestor"/>.
     /// </summary>
-    public bool DontScanErrorPage { get; private set; }
+    public bool DontScanErrorPage { get; }
```

Comment:
The setter should rather be public, because otherwise this property doesn't make sense.

Reply:
It can't be public as far as the property itself is public as needs to be less accessbile. The best solution in my opinion here is to reconsider this whole rule in these cases. Follow up here https://github.com/Lombiq/UI-Testing-Toolbox/pull/378#discussion_r1634748833

Reply:
Oh yes. My bad. You meant `{ get; set;}` and not `{ get; public set; }`. Brainmelt on my side.